pontoon/teams/static/js/bugzilla.js

```diff
@@ -65,6 +65,16 @@ var Pontoon = (function (my) {
                   html: formatter.format(new Date(bug.creation_time))
                 }).appendTo(tr);
 
+                $('<td>', {
+                  class: 'assignee',
+                  html: formatter.format(new Date(bug.assignee))
+                }).appendTo(tr);
+  
+                $('<td>', {
+                  class: 'status',
+                  html: formatter.format(new Date(bug.status))
+                }).appendTo(tr);
```

Comment:
After a sound thought, I think we should not add the `status` field, as it's mostly going to be the same. The reporter also hinted at that:
https://bugzilla.mozilla.org/show_bug.cgi?id=1423854#c0

So let's remove that.

Reply:
Okay. That's true